content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,70 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myconfigmap
+data:
+  username: "k8s-admin"
+  access_level: "1"
+```
+
+The following command will create the ConfigMap object:
+
+```shell
+kubectl apply -f myconfigmap.yaml
+configmap/myconfigmap created
+```
+
+The following command will produce a JSON object that contains the ConfigMap's key-value pairs:
+
+```shell
+kubectl get configmap myconfigmap -o jsonpath='{.data}' | jq
+{
+  username: "k8s-admin"
+  access_level: "1"
+}
```

Comment:
We [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)